### PR TITLE
723 project kick off changes to make in build before private beta

### PR DESCRIPTION
**Pull request by titlescreen**
## Changes

- Update content on Handover to regional delivery officer
- Update content on external stakeholder kick-off
- Update content on process conversion grant

Content checked with @SteveMilnes. 
Needs a dev to check code 

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Comment by titlescreen** [comment]
I tried running prettier to fix the formatting error on kickoff.yml, but it failed and did not make any changes or tell me where the error was.

File: app/workflows/lists/conversion/sections/kickoff.yml [hunk 1]
```diff
@@ -1,53 +1,49 @@
...
-          Make notes and write questions to as the regional delivery officer
-        hint: |
+          Make notes and write questions to ask the regional delivery officer
+        guidance_summary: What to make notes about
+        guidance_text: |
           Note down things you want to ask and talk about with the regional delivery officer at the handover meeting.
 
           This is an opportunity to clarify anything that you're unsure about after reviewing the project and doing your background research.
       - title: Attend handover meeting with regional delivery officer
         hint: |
```
Comment (jkempster34) [inline app/workflows/lists/conversion/sections/kickoff.yml:35]: Can this pipe (`|`) go now that it's no longer needed?

File: app/workflows/lists/conversion/sections/kickoff.yml [hunk 2]
```diff
@@ -138,39 +135,36 @@ tasks:
 
       - title: Check and confirm the school's grant eligibility
         hint: |
```
Comment (jkempster34) [inline app/workflows/lists/conversion/sections/kickoff.yml:137]: Another pipe that can go

File: app/workflows/lists/conversion/sections/kickoff.yml [hunk 3]
```diff
@@ -138,39 +135,36 @@ tasks:
...
+          Check in the school's SharePoint folder to see if a Grant claim form has been submitted before.
 
-          Check in the school SharePoint folder to see if a Grant claim form has been submitted already.
+          You can also email the Grant payments team at [CFUGrantPayments.REGIONSGROUP@education.gov.uk](mailto:CFUGrantPayments.REGIONSGROUP@education.gov.uk) to check if the school has previously received any money, and how much they're still eligible for.
 
-          You can also email the Grant payments team at [CFUGrantPayments.REGIONSGROUP@education.gov.uk](mailto:CFUGrantPayments.REGIONSGROUP@education.gov.uk) to check if the school has previously received any money, and how much they’re still eligible for.
+          You must include this information in your email to the Grant payments team later.
 
       - title: Receive, check and save the Grant payment form
         hint: |
```
Comment (jkempster34) [inline app/workflows/lists/conversion/sections/kickoff.yml:152]: Pipe

**Review by jkempster34 — CHANGES_REQUESTED** [review]

**Review by jkempster34 — APPROVED** [review]
🚢

**Pull request merged**